### PR TITLE
Added recommendation 

### PR DESCRIPTION
**Pull request by unnati914**
solves #19 
image

### CODE REVIEW

File: .eslintcache [hunk 1]
```diff
@@ -0,0 +1 @@
...umn":16,"fix":"64"},{"ruleId":"50","severity":1,"message":"65","line":72,"column":28,"nodeType":null,"messageId":"66","endLine":72,"endColumn":28,"fix":"67"},{"ruleId":"50","severity":1,"message":"68","line":121,"column":22,"nodeType":null,"messageId":"66","endLine":121,"endColumn":22,"fix":"69"},"prettier/prettier","Delete `······`","delete",{"range":"70","text":"71"},"Delete `··`",{"range":"72","text":"71"},"Replace `··<span>The·weather·looks·clear,·you·can·go·out!·` with `<span>The·weather·looks·clear,·you·can·go·out!`","replace",{"range":"73","text":"74"},"Delete `·`",{"range":"75","text":"71"},"Replace `··············<span>Please·take·an·umberlla!··` with `················<span>Please·take·an·umberlla!·`",{"range":"76","text":"77"},"Replace `(⏎················<span>Stay·inside</span>⏎··············)` with `<span>Stay·inside</span>`",{"range":"78","text":"79"},"Insert `⏎`","insert",{"range":"80","text":"81"},"Insert `;␍⏎`",{"range":"82","text":"83"},[2331,2337],"",[2374,2376],[676,725],"<span>The weather looks clear, you can go out!",[866,867],[938,984],"                <span>Please take an umberlla! ",[1179,1237],"<span>Stay inside</span>",[2466,2466],"\n",[2583,2583],";\r\n"]
```
Comment (yashika51) [inline .eslintcache:1]: Delete this file :)

**Review by yashika51 — CHANGES_REQUESTED** [review]

**Review by PAOK-2001 — CHANGES_REQUESTED** [review]
Cool idea! But I don't think it looks good on the weather card itself, I would create another space fot the recomendation. Or perhaps bellowe the bottom info on the card.

**Comment by unnati914** [comment]
maybe its minor css changes @PAOK-2001 , if we want it on weather card

**Comment by PAOK-2001** [comment]
@unnati914 it seems this changes got pushed in your other PR. I'll try and move the style and I'll make a PR

**Comment by yashika51** [comment]
@unnati914 , @PAOK-2001 can this PR be closed?

**Comment by unnati914** [comment]
yes, ig we can create a new PR for this

**Comment by unnati914** [comment]
closing this

**Comment by Mayank17M** [comment]
#52 fixes this

**Pull request closed without merging**